sihl-queue/src/sihl_queue.ml

```diff
@@ -192,14 +201,53 @@ module Make (Repo : Repo.Sig) : Sihl.Contract.Queue.Sig = struct
   let register ?(jobs = []) () =
     Repo.register_migration ();
     Repo.register_cleaner ();
-    let jobs_to_register = jobs |> List.map Workable_job.of_job in
-    registered_jobs := List.concat [ !registered_jobs; jobs_to_register ];
+    registered_jobs := List.concat [ !registered_jobs; jobs ];
     Sihl.Container.Service.create lifecycle
   ;;
+
+  let query () : Instance.t list Lwt.t = Repo.query ()
+
+  let find id : Instance.t Lwt.t =
+    let open Lwt.Syntax in
+    let* job = Repo.find id in
+    match job with
+    | Some job -> Lwt.return job
+    | None ->
+      raise @@ Exception (Format.asprintf "Failed to find with id %s" id)
+  ;;
+
+  (* let delete (job : Instance.t) : unit Lwt.t = Repo.delete job *)
```

Comment:
Remove please

Reply:
@jerben You missed this one. (GitHub hid it and you have to expand manually, pretty annoying)

Reply:
Fixed.